packages/docs-app/src/examples/popover2-examples/popover2Example.tsx

```diff
@@ -0,0 +1,346 @@
+/*
+ * Copyright 2021 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Placement, placements as PLACEMENT_OPTIONS } from "@popperjs/core";
```

Comment:
corresponding to above comment... these should both come from the popover2 package. `PopperPlacement` is already exported.